config.md

```diff
@@ -52,7 +51,7 @@ For Solaris, the mounts corresponds to fs resource in zonecfg(8).
   For the Solaris operating system, this corresponds to "dir" of the fs resource in zonecfg(8).
 * **`type`** (string, REQUIRED) The filesystem type of the filesystem to be mounted.
   Linux: *filesystemtype* argument supported by the kernel are listed in */proc/filesystems* (e.g., "minix", "ext2", "ext3", "jfs", "xfs", "reiserfs", "msdos", "proc", "nfs", "iso9660").
-  Windows: ntfs.
+  Windows: corresponds to the type of file system on the volume, e.g. 'ntfs'.
```

Comment:
I thought using an example rather than implying only 'ntfs' would be better. I pinged irc for supported file systems but no reply yet. It's possible that only 'ntfs' is supported, but I thought we might see 'refs' eventually.

@RobDolinMS could you advise what should go here?